python/modules/Stop0l_trigger.py

```diff
@@ -256,15 +279,15 @@ def analyze(self, event):
 	MET_trigger_eff_low_dm = MET_trigger_eff_low_dm_down = MET_trigger_eff_low_dm_up = 0
 	MET_trigger_eff_loose_baseline_QCD = MET_trigger_eff_loose_baseline_QCD_down = MET_trigger_eff_loose_baseline_QCD_up = 0
 	MET_trigger_eff_high_dm_QCD = MET_trigger_eff_high_dm_QCD_down = MET_trigger_eff_high_dm_QCD_up = 0
-	MET_trigger_eff_low_dm_QCD = MET_trigger_eff_low_dm_QCD_down = MET_trigger_eff_low_dm_QCD_up = 0
+	MET_trigger_eff_low_dm_QCD = MET_trigger_eff_low_dm_QCD_down = MET_trigger_eff_low_dm_QCD_up = MET_trigger_eff_low_dm_QCD_down_METSig = MET_trigger_eff_low_dm_QCD_up_METSig = 0
 
```

Comment:
These long line of assignment could be dangerous. I believe python will consider they reference to same variable. If one of them was assigned to one value, the other will assigned same value. Please double check